.github/workflows/ci.yml

```diff
@@ -96,3 +96,12 @@ jobs:
         run: docker build -t ubuntucontainer tss-esapi/tests/ --file tss-esapi/tests/Dockerfile-ubuntu
       - name: Check documentation
         run: docker run -v $(pwd):/tmp/rust-tss-esapi -w /tmp/rust-tss-esapi/tss-esapi -e RUSTDOCFLAGS="-Dwarnings" ubuntucontainer cargo doc --document-private-items --no-deps
+
+  # Check that there are no Clippy lint errors.
+  clippy:
+    name: Check Clippy lints
```

Comment:
I'm not sure if this solves the problem without `continue on error`: https://stackoverflow.com/a/73357322

Reply:
No I do not think it either because I will probably need to run in a container. Because it is run with all features which means that it wants to generate bindings. I am not sure that we want that?

Reply:
I mean that without `continue-on-error: true` failing clippy will abort all other jobs and as I understood in #438 @ionut-arm wanted to continue even on clippy failure. Thus "This fixes https://github.com/parallaxsecond/rust-tss-esapi/issues/468." is not correct IMHO.

Reply:
Oh, I see.. I thought it would continue running the others regardless. That would be ideal, but even if we just avoid everything failing because of clippy would also be good.

Reply:
Are you sure it will abort all other jobs? I thought it would only abort the remaining steps in the same job.

Reply:
Hmm... I *was* sure when I wrote my comment as I had a similar issue elsewhere but now I'm double thinking it... I'll need to run a test or two to verify...

Reply:
Yep, you're right. I just verified that and it runs other jobs to completion even if the fastest one fails: https://github.com/wiktor-k/ci-test/actions/runs/7567980145

Sorry for the noise, resolving this discussion!